src/exts/mod.py

```diff
@@ -46,9 +47,9 @@ async def mod(
                 case "channel":
                     match sub_command:
                         case "slowmode":
-                            await self._slowmode_channel(ctx, channel, length)
+                            await self._slowmode_channel(ctx, length, channel)
```

Comment:
I can't find this method?

Reply:
It is not implemented yet. I just change it to match with the required option schema from Discord.

The problem with that is, setting ``set_rate_limit_per_user`` directly will override all permission of the channel. As this requires more like from the moderation side to understand and manually set up the ``modify()``, there is nothing much I can do though.

Reply:
oh ok well we'll ignore it then